app/views/book/show.html.erb

```diff
@@ -16,9 +16,9 @@
     <%= @book.description_html %>
   </div>
 
-  <% if @book.next_exercise_for(current_user).present? %>
+  <% @book.next_lesson_for(current_user).try do |lesson| %>
```

Comment:
@aguspina then I realized than using try here was better than using the instance variable AND calling the method twice